src/features/create-dao/Steps/DetailsForm.test.tsx

```diff
@@ -62,9 +62,7 @@ describe('<DetailsForm />', () => {
 		);
 
 		await waitFor(() =>
-			expect(
-				screen.getByText('The name field is required.'),
-			).not.toBeInTheDocument(),
+			expect(screen.getByText('The name field is required.')).toBeVisible(),
```

Comment:
I'm not sure if the test description aligns with the expectation. Should the input validation error be visible if the required name field value is valid?

Could be wrong but looks like the test should be described as something like `should throw invalid input error`. I might just be reading this wrong!

Same for the tests below mate